api_tests/wb/views/test_wb_hooks.py

```diff
@@ -802,6 +840,43 @@ def test_invalid_payload(self, app, copy_url):
         assert res.status_code == 400
         assert res.json['errors'][0]['detail'] == 'Invalid Payload'
 
+    def test_storage_usage_move_within_node(self, app, node, signed_payload, copy_url):
+        """
+        Checking copys within a node, since the net size of the node hasn't changed the cache will remain expired at None
+        """
+        assert node.storage_usage is None
+
+        res = app.post_json(copy_url, signed_payload)
+
+        assert res.status_code == 201
+        assert node.storage_usage is None  # this is intentional the cache shouldn't be touched
+
+    def test_storage_usage_move_between_nodes(self, app, node, node_two, file, user, node_two_root_node, copy_url):
+        """
+        Checking storage usage when copying files to outside a node means only the destination should be recalculated.
+        """
+
+        assert node.storage_usage is None  # the cache starts expired, but there is 1337 bytes in there
+        assert node_two.storage_usage is None  # zero bytes here
+
+        signed_payload = sign_payload(
+            {
+                'source': file._id,
+                'target': node._id,
+                'user': user._id,
+                'destination': {
+                    'parent': node_two_root_node._id,
+                    'target': node_two._id,
+                    'name': 'test_file',
+                }
+            }
+        )
+        res = app.post_json(copy_url, signed_payload)
+        assert res.status_code == 201
+
+        assert node.storage_usage is None
```

Comment:
Something I'd like to see in this test is asserting that node.storage_usage is also 1337 - this was a copy, not a move.

Reply:
I've added a better check below: https://github.com/CenterForOpenScience/osf.io/pull/9091/files#diff-b54bb3980acb4d40f7f429872ee0c68dR882-R884